### PR TITLE
@craigspaeth: Remove shrinkwrap in favor of explicit versions & < 80 characters

### PR DESCRIPTION
**Pull request by craigspaeth**
Besides the sub-dependencies this doesn't bring anything new functionality-wise. I've been debugging Facebook signup all day and all of the sudden some state I'm in locally is now working. So I thought might as well bring this close to the same state.

FWIW I'm pretty sure the `state: true` param is causing a "Forbidden" on Facebook signup currently.

### CODE REVIEW

File: index.coffee [hunk 1]
```diff
@@ -63,7 +67,10 @@ initApp = ->
 initPassport = ->
   passport.serializeUser serializeUser
   passport.deserializeUser deserializeUser
-  passport.use new LocalStrategy { usernameField: 'email', passReqToCallback: true }, artsyCallback
+  passport.use new LocalStrategy(
+    { usernameField: 'email', passReqToCallback: true }
+    artsyCallback
```
Comment (broskoski) [inline index.coffee]: will this pass as a second argument without a comma?

**Comment by broskoski** [comment]
:clap:

File: index.coffee (hunk 1 above)
Comment (craigspaeth) [inline index.coffee]: Sure will!

Comment (broskoski) [inline index.coffee]: :+1: never knew that!

Comment (craigspaeth) [inline index.coffee]: These are all just breaking before 80 characters btw... nothing functionally should have changed.

**Pull request merged**